### ISSUE TITLE
Descriptions for multiple tags aren't present in example or spec

### ISSUE THREAD

**Issue by talexand** [fotinakis/swagger-blocks#23]
I'm having trouble making tag descriptions for multiple tags. Both https://github.com/fotinakis/swagger-blocks/blob/master/spec/lib/swagger_v2_blocks_spec.rb and the README don't show how to do this, but I tried just listing them in order like

```ruby
    tags do
      key :name, 'article'
      key :description, 'Longform articles'
      externalDocs do
        key :description, 'Find more info here'
        key :url, 'https://swagger.io'
      end
      key :name, 'event_mod_action'
      key :description, 'Moderation actions on events'
      externalDocs do
        key :description, 'Find more info here'
        key :url, 'https://swagger.io'
      end
      key :name, 'event'
      key :description, 'Community events'
      externalDocs do
        key :description, 'Find more info here'
        key :url, 'https://swagger.io'
      end
```

as well as placing the properties in a block like 

```ruby
    tags do
      key :name, 'article' do
        key :description, 'Longform articles'
        externalDocs do
          key :description, 'Find more info here'
          key :url, 'https://swagger.io'
        end
```

with no success. What's the right way to do this?

**Comment by fotinakis** [comment]
Glanced at the code, pretty sure if you do three different `tags do ... end` blocks it will append them. So, just your first example, but with three different tags blocks.

**Comment by talexand** [comment]
Oh ok. So `tags` blocks refers to a single tag? That's a little bit confusing when `parameter` and `operation` and `response` are singular for their respective blocks.

**Comment by fotinakis** [comment]
True, just fixed in https://github.com/fotinakis/swagger-blocks/commit/b01ef2b01dfd8f15c4e8f483c9a52d5445207926 and released as v1.1.3.
